src/ydb.cc

```diff
@@ -3063,26 +3063,29 @@ env_dbremove_subdb(DB_ENV * env, DB_TXN * txn, const char *fname, const char *db
 //          open, close, and begin checkpoint cannot occur.
 // returns: true if we could open, lock, and close a dictionary
 //          with the given dname, false otherwise.
-static bool
+static int
```

Comment:
Since you changed the return type, can you also update the description comment above.